### ISSUE TITLE
Temporarily allow the same logins to be used across different graphs

### ISSUE THREAD

**Issue by benloh** [netcreateorg/netcreate-itest#233]
For the september pilot, we expect students to be working with multiple graphs over the duration of the pilot.

To make this easier to manage, it'd be useful to have a single login token for each student that can be used across multiple graphs.

Looking at the `MakeToken` call in `comment-session.ts`, it appears that we use `dataset` to salt the hash.
If we simply removed `dataset` as a salt, would tokens still work?  e.g. instead of this:
```
128:   let salt = `${classId}${projId}${dataset}`;
```
simplify it as:
```
128:   let salt = `${classId}${projId}`;
```
Is that potentially one way to hand out tokens that can work across multiple graphs?

# To Do
* [ ] Update `nc-multiplex` to support dataset-agnostic tokens

**Comment by daveseah** [comment]
Yup, I think that should work. So long as the multiple graphs do not have unique identifying elements that prevent them from being cross-used, I think it will be OK.

**Comment by daveseah** [comment]
There is a corresponding DecodeToken() method too of course, or something like that.

**Comment by benloh** [comment]
Implemented with #241.
